rfcs/0000_template.md

```diff
@@ -1,25 +1,46 @@
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: (leave this empty)
-- Kibana Issue: (leave this empty)
+- TTL: (e.g. "April 20th, 2021", time the review is expected to be completed by. Don't use relative days.)
+- Champion: (usually you, person who writes and updates the draft and incorporates feedback)
+- Main reviewer: (somebody familiar with the subject matter, who has committed to provide timely and detailed reviews for this RFC)
+- Owner team: (team who will own this feature, if it is accepted)
+- Stakeholders: (people or groups who may be affected by the proposed changes)
+- RFC PR: (link to PR which implements a PoC of the proposal)
```

Comment:
The purpose of this field was changed, and the title "RFC PR" is no longer accurate. Historically, this was a link to the RFC PR itself. For example, https://github.com/elastic/kibana/blob/master/rfcs/text/0012_encryption_key_rotation.md has a "RFC PR" linked to https://github.com/elastic/kibana/pull/72828.

I think we should leave the "RFC PR" field and provide a better description, in addition to adding a new "POC PR" field.

Reply:
Done.